browser/pages/about/controller.js

```diff
@@ -10,12 +10,16 @@ class AboutController {
     $scope.version = electron.remote.app.getVersion();
   }
 
-  next() {
-    this.router.go('location');
+  documentation() {
+    this.electron.shell.openExternal('https://access.redhat.com/documentation/en/red-hat-development-suite/');
   }
 
-  openDevSuiteOverview() {
-    this.electron.shell.openExternal('https://developers.redhat.com');
+  release() {
+    this.electron.shell.openExternal('https://access.redhat.com/documentation/en-us/red_hat_development_suite/1.4/html/release_notes_and_known_issues/');
```

Comment:
The installer version should not be hardcoded into the link. Use https://www.npmjs.com/package/semver module functions:
* major(version)
* minor(version)
to generate right url like
``` javascript
get shortVersion() {
  return `${semver.major($scope.version)}.${semver($scope.version).minor()}`;
}

release() {
  this.electron.shell.openExternal(`https://access.redhat.com/documentation/en-us/red_hat_development_suite/${this.shortVersion}/html/release_notes_and_known_issues/`);
}
```